version.php

```diff
@@ -26,6 +26,6 @@
 
 $plugin->component = 'quizaccess_proctoring';
 $plugin->release = '1.1.0';
```

Comment:
Make it 1.1.1

Reply:
made the release version 1.1.1 as requested.